Client/RecommendationClient.php

```diff
@@ -22,4 +22,13 @@ public function updateContent($contentId);
      * @return void
      */
     public function deleteContent($contentId);
+
+    /**
+     * @param int $userId YooChoose user ID
+     * @param int $scenarioId YooChoose scenario ID
```

Comment:
@bdunogier I suggested something different. I wanted to see some sample values, like `userId` might look like: `u12434` while `scenarioId` should look like `s234`. It would be useful in the future when someone is back to this functionality with a will to modify it.